### PR TITLE
{phys}[gompi/2022a] fastjet-contrib v1.049

### PR DESCRIPTION
**Pull request by APN-Pucky**
(created using `eb --new-pr`)

Requires:
- [ ] https://github.com/easybuilders/easybuild-easyconfigs/pull/17367

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/99506f0dda630f6be95499fa0c75c4d2 for a full test report.

File: easybuild/easyconfigs/f/fastjet-contrib/fastjet-contrib-1.049-gompi-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+source_urls = ['https://fastjet.hepforge.org/contrib/downloads/']
+sources = ['fjcontrib-' + version + '.tar.gz']
+checksums = ['ae2ed6206bc6278b65e99a4f78df0eeb2911f301a28fb57b50c573c0d5869987']
+
+dependencies = [
+    ('fastjet', '3.4.0')
+]
+
+configopts = ""
```
Comment (verdurin) [inline easybuild/easyconfigs/f/fastjet-contrib/fastjet-contrib-1.049-gompi-2022a.eb:24]: @APN-Pucky could this be removed?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/fastjet-contrib/fastjet-contrib-1.049-gompi-2022a.eb (hunk 1 above)
Comment (APN-Pucky) [reply easybuild/easyconfigs/f/fastjet-contrib/fastjet-contrib-1.049-gompi-2022a.eb:24]: Yes just checked that.

Comment (verdurin) [reply easybuild/easyconfigs/f/fastjet-contrib/fastjet-contrib-1.049-gompi-2022a.eb]: Thanks.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/81b63d1178d59082587d786d49d07727 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @APN-Pucky!

**Pull request merged**